tox.ini

```diff
@@ -2,19 +2,25 @@
 # Jython 2.7rc2 does work, but unfortunately has an issue running
 # with Tox 1.9.2 (http://bugs.jython.org/issue2325)
 #envlist = py26,py27,py33,py34,pypy,simple,jython,pypy3
-envlist = py27,py33,py34,py35,pypy,simple,pypy3
+envlist = py27,py34,py35,py36,pypy,simple,pypy3
 
 [testenv]
+# ZODB.tests.testdocumentation needs to find
+# itself in the source tree to locate the doc/
+# directory. 'usedevelop' is more like what
+# buildout.cfg does, and is simpler than having
+# testdocumentation.py also understand how to climb
+# out of the tox site-packages.
+usedevelop = true
 commands =
 # Run unit tests first.
-    zope-testrunner -u --test-path=src --auto-color --auto-progress
+    zope-testrunner -u --test-path=src []
 # Only run functional tests if unit tests pass.
-    zope-testrunner -f --test-path=src --auto-color --auto-progress
+    zope-testrunner -f --test-path=src []
 # without explicit deps, setup.py test will download a bunch of eggs into $PWD
```

Comment:
Minor: opportunity to drop this outdated comment, since we're not using `setup.py test`